docker-app/qfieldcloud/subscription/models.py

```diff
@@ -595,14 +595,14 @@ def set_package_quantity(
         return old_package, new_package
 
     @classmethod
-    def get_or_create_active_subscription(cls, account: UserAccount) -> "Subscription":
-        """Returns the currently active subscription, if not exists returns a newly created subscription with the default plan.
+    def get_or_create_current_subscription(cls, account: UserAccount) -> "Subscription":
+        """Returns the current subscription, if not exists returns a newly created subscription with the default plan.
 
         Args:
             account (UserAccount): the account the subscription belongs to.
 
         Returns:
-            Self: the currently active subscription
+            Self: the current subscription
 
```

Comment:
IMO talking just of the `current subscription` works, thus we could also use `currently running subscription` ... ?